workshops/templates/base.html

```diff
@@ -40,6 +40,15 @@
                 {% navbar_element "Search" "search" %}
               </ul>
             </li>
+            <li class="dropdown">
+              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Reports <span class="caret"></span></a>
+              <ul class="dropdown-menu" role="menu">
+                {% navbar_element "Problems" "problems" %}
+                {% navbar_element "Workshops over time" "workshops_over_time" %}
+                {% navbar_element "Learners over time" "learners_over_time" %}
+                {% navbar_element "Instructors over time" "instructors_over_time" %}
+	      </ul>
+	    </li>
```

Comment:
If this is to be visited rarely and only by you, then I'd move it into separate section in "More" (and have "Problems" change name to "Report problems").